Index: vendor/inacho/php-credit-card-validator/src/CreditCard.php
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/vendor/inacho/php-credit-card-validator/src/CreditCard.php b/vendor/inacho/php-credit-card-validator/src/CreditCard.php
--- a/vendor/inacho/php-credit-card-validator/src/CreditCard.php
+++ b/vendor/inacho/php-credit-card-validator/src/CreditCard.php	(date 1688020394732)
@@ -206,12 +206,12 @@
     {
         $checksum = 0;
         for ($i=(2-(strlen($number) % 2)); $i<=strlen($number); $i+=2) {
-            $checksum += (int) ($number{$i-1});
+            $checksum += (int) ($number[$i-1]);
         }

         // Analyze odd digits in even length strings or even digits in odd length strings.
         for ($i=(strlen($number)% 2) + 1; $i<strlen($number); $i+=2) {
-            $digit = (int) ($number{$i-1}) * 2;
+            $digit = (int) ($number[$i-1]) * 2;
             if ($digit < 10) {
                 $checksum += $digit;
             } else {
